### PR TITLE
updates tables

### PR DESCRIPTION
**Pull request by DHowell1150**
Took any pr date and weight out of Movements and put it into ProgressMovements
Added image and video to movements as strings
deleted movement_name from Goals

### CODE REVIEW

File: db/migrate/20240517155942_create_user.rb
Comment (s2an) [inline db/migrate/20240517155942_create_user.rb:1]: Nice! When you move to the user authentication lesson, I think you will need to add a :password_digest column, but the lesson goes over that. The user authentication lesson would be best to do next (After all the basic validation tests for the models!). That will show how to use bcrypt and do the :has_secure_password confirmation callback.

File: db/migrate/20240517161522_create_goals.rb
Comment (s2an) [inline db/migrate/20240517161522_create_goals.rb:1]: Seeing this makes me wonder if the goals could be linked to the movement as well... I'll need to think on that, but we won't need to make goals for a while yet (not until after the user auth and some API calls).

File: db/migrate/20240517165834_create_movement_goals.rb
Comment (s2an) [inline db/migrate/20240517165834_create_movement_goals.rb:1]: I wonder if the weight and reps would go here (like the progress_movements table. Again, we have time to look into that later.

**Comment by s2an** [comment]
I think this is a great start! I'll go ahead and merge it so you can work on user auth, generating/creating models, controllers, basic views, and tests. For now, just worry about user, athlete, movement, progress, and movement_progress and their tests. For user: we will need show and create/new. For athlete, movement, and progress we will need index, show, create/new, and edit I believe.

File: db/migrate/20240517161522_create_goals.rb
Comment (DHowell1150) [reply db/migrate/20240517161522_create_goals.rb:1]: I did create a relationship has_many :movements, through: :movement_goals.  Is that what you mean?

**Pull request merged**